clang/unittests/Format/FormatTestJS.cpp

```diff
@@ -834,6 +834,11 @@ TEST_F(FormatTestJS, AsyncFunctions) {
                "}",
                "async function hello(myparamnameiswaytooloooong) {}",
                getGoogleJSStyleWithColumns(10));
+  verifyFormat("async function\n"
+               "union(\n"
+               "    myparamnameiswaytooloooong) {\n"
+               "}",
+               getGoogleJSStyleWithColumns(10));
```

Comment:
Can you add something like `const auto Style = getGoogleJSStyleWithColumns(10);` and reuse it here and at line 849 below?